packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -34,7 +39,14 @@ class ScrollPhysics {
   /// If a subclass of [ScrollPhysics] does not override a method, that subclass
   /// will inherit an implementation from this base class that defers to
   /// [parent]. This mechanism lets you assemble novel combinations of
-  /// [ScrollPhysics] subclasses at runtime.
+  /// [ScrollPhysics] subclasses at runtime. For example:
+  ///
+  /// ```dart
+  /// BouncingScrollPhysics(parent: AlwaysScrollPhysics())
+  ///
+  /// ```
+  /// will result in a [ScrollPhysics] that has the combined behavior
```

Comment:
Just saying 'combined' is a bit oversimplifying it. We should say something to the effect of 'behaviors not specified by BouncingScrollPhysics will defer to AlwaysScrollPhysics'.